tests/test_ingest.py

```diff
@@ -42,25 +42,29 @@ class HscIngestTestCase(IngestTestBase, lsst.utils.tests.TestCase):
     curatedCalibrationDatasetTypes = ("defects", "camera", "bfKernel", "transmission_optics",
                                       "transmission_sensor", "transmission_filter", "transmission_atmosphere")
 
-    def setUp(self):
-        self.ingestDir = os.path.dirname(__file__)
-        self.instrument = "lsst.obs.subaru.HyperSuprimeCam"
-        self.instrumentName = "HSC"
-        self.file = os.path.join(testDataDirectory, "hsc", "raw", "HSCA90402512.fits.gz")
-        self.dataIds = [dict(instrument="HSC", exposure=904024, detector=50)]
-
-        super().setUp()
-        self.butler = Butler(self.root, run=self.outputRun)
-        self.visits = {
+    ingestDir = os.path.dirname(__file__)
+
+    instrumentClassName = "lsst.obs.subaru.HyperSuprimeCam"
+
+    instrumentName = "HSC"
+
+    file = os.path.join(testDataDirectory, "hsc", "raw", "HSCA90402512.fits.gz")
```

Comment:
This .gz reminds me that I think we want to change the default for ingest to also include `.fits.gz` and `.fits.fz`.

Reply:
@timj did you want me to do something about this in this ticket? (FWIW if it involves more than a line or 2 of code I'd rather ticket it)

Reply:
It was one line in python/lsst/obs/base/script/ingestRaws.py where we check file extension and could include the .gz/.fz there directly -- the ingestion clearly supports .fits.gz because the above file is a .fits.gz.  What I'm now wondering is why we are specifying a file here at all when it seems like we could specify the directory -- maybe this is deliberate to ensure that we know exactly which dataIds we will end up with in the butler.

Reply:
We should take files as any number of extra positional arguments and let the shell do the globbing for us.